content/posts/newsletter-015/index.md

```diff
@@ -668,6 +668,47 @@ on the module's [repository][Proton-Media-Converter-Github]
 [Proton-Github]: https://github.com/ValveSoftware/Proton
 [Proton-Media-Converter-Github]: https://github.com/ValveSoftware/Proton/tree/proton_5.13/media-converter
 
+### [Rust GPU][RustGPU-Github] v0.1
+
+![Rust GPU Sky](rustgpu.jpg)
+_Sky example in Rust GPU_
+
+Rust GPU is a very early stage project to make Rust a first-class language and
```

Comment:
I think I would rather restructure this into something like:

Rust GPU is a project backed by [Embark Studios][RustGPU-Embark] to make Rust a first-class language and ecosystem for building GPU code.

[RustGPU-Embark]: https://www.embark-studios.com/

Reply:
:ok_hand: